src/Core/Packages/ZipPackage.cs

```diff
@@ -115,6 +117,18 @@ public override void ExtractContents(IFileSystem fileSystem, string extractPath)
             }
         }
 
+        public static string GetPackageIdentifier(Package package)

```

Comment:
Add a comment that this may be null, and explain how the xml exception can happen in the code